scripts/exwafs_grib.sh

```diff
@@ -44,6 +44,8 @@ set -x
 
 cd "${DATA}" || err_exit "FATAL ERROR: Could not 'cd ${DATA}'; ABORT!"
 
+export jobsuffix="gfs_atmos_wafs_f${fhr}_$cyc"
```

Comment:
@aerorahul This line has to be moved from ush to script: jobsuffix="gfs_atmos_wafs_f${fhr}_$cyc", because fhr doesn't have the same value in these two scripts.